public/js/main.js

```diff
@@ -57,18 +64,11 @@ if (connTarget) {
     setTabTarget: firefox.setTabTarget,
     initPage: firefox.initPage,
     getBoundActions: () => boundActions,
-    renderApp: () => renderRoot(store, App)
+    renderApp: () => renderRoot(App)
   };
 } else {
   connectClients().then(tabs => {
-    const mount = document.querySelector("#mount");
-
-    // bail in test environments that do not have a mount
-    if (!mount) {
-      return;
-    }
-
     boundActions.newTabs(tabs);
-    renderRoot(store, Tabs, mount);
+    renderRoot(Tabs);
```

Comment:
I didn't see the point of passing those things in, and when we move to v3 of react-hot-loader it'll make it easier (we can just call renderRoot again).